src/components/views/beta/BetaCard.tsx

```diff
@@ -100,6 +100,14 @@ const BetaCard = ({ title: titleOverride, featureId }: IProps) => {
         </AccessibleButton>;
     }
 
+    let refreshWarning: string;
+    if (requiresRefresh) {
```

Comment:
Could you set `requiresRefresh: true` for threads whilst you're making it do more things, thanks